common/test/acceptance/pages/lms/dashboard.py

```diff
@@ -177,3 +177,16 @@ def pre_requisite_message_displayed(self):
         Verify if pre-requisite course messages are being displayed.
         """
         return self.q(css='section.prerequisites > .tip').visible
+
+    def click_username_dropdown(self):
+        """Click username dropdown"""
+        self.q(css='.dropdown').first.click()
+
+    @property
+    def username_dropdown_links(self):
+        """Return list username dropdown links"""
```

Comment:
Can we add the doc string like 
```
"""
Return list username dropdown links
"""
```
Please also change for `click_account_settings_link` and `click_username_dropdown` methods.